botocore/httpchecksum.py

```diff
@@ -248,16 +253,18 @@ def resolve_request_checksum_algorithm(
         # request supports it, use that instead of checksum required
         if supported_algorithms is None:
             supported_algorithms = _SUPPORTED_CHECKSUM_ALGORITHMS
+        if crt_supported_algorithms is None:
+            crt_supported_algorithms = _CRT_SUPPORTED_CHECKSUM_ALGORITHMS
```

Comment:
I don't think this needs to be passed in, we can use the static list. There isn't a strong reason for it to be dynamic/adjustable per invocation like supported checksums.